File: Domain/Ticket.cs
namespace Domain {
    public class Ticket
    {
    public Guid Id { get; set; } 
    public Guid CustomerId { get; set; } 
    public Customer Customer { get; set; } 
    public Guid ActivityId { get; set; } 
    public Activity Activity { get; set; } 
    public Guid TicketSeatId { get; set; } 
    public TicketSeat TicketSeat { get; set; } 
    }
}

File: client-app/src/models/ticket.ts
import { Activity } from "./activity";
import { Customer } from "./customer";
import { TicketSeat } from "./ticketSeat";

export interface Ticket {
    id?: string;
    activity: Activity,
    activityId?: string;
    customer: Customer;
    customerId?: string;
    ticketSeat: TicketSeat;
    ticketSeatId?: string;
}